practice/pyramid/site.css

```diff
@@ -0,0 +1,26 @@
+div {
+  border: 1px solid black;
+}
+
+.row {
+  margin: 20px;
+  display: flex;
+  height: 100px;
```

Comment:
Set widths from the top down and heights from the bottom up. So innermost divs should have explicit heights, not outermost ones.